actions/dump-charm-artifacts/logdump.bash

```diff
@@ -0,0 +1,61 @@
+# Dumps logs relevant to kubernetes-based Juju charms
+# Developed for dumping logs from GH Workflows, but useful for other places too
+# 
+# Logs are both printed to screen and dumped in the OUTPUT_DIR (default: tmp)
+#
+# Prerequisites:
+# * juju crashdump (`snap install juju-crashdump`)
+# * kubectl (with a kube config for the relevant cluster)
+# * ketall (https://github.com/corneliusweig/ketall)
+#
+# Warning: This dumps a lot of details from your kubernetes cluster and juju models.
+# Before sending this information to anyone, you should inspect it to ensure no
+# sensitive information is being shared.
+
+
+# Inputs
+OUTPUT_DIR=${OUTPUT_DIR:-tmp}
+# Defaults to the location on a gh runner
+CHARMCRAFT_LOG_DIR=${CHARMCRAFT_LOG_DIR:-"/home/runner/snap/charmcraft/common/cache/charmcraft/log"}
```

Comment:
how am I supposed to run this script locally ?

Reply:
`CHARMCRAFT_LOG_DIR=your/log/dir bash logdump.bash` should do it.  These are just environment variables with a default that works on the runner

Reply:
should we cover it in readme ?

should we change it to `$USER/snap/....`
so, it will work on local machines as well?

Reply:
does `$HOME/snap` work on gh runners?  I swear it doesn't... I'll check

Reply:
Added to readme, and will test whether I can use $HOME in a second commit